automation_tools/satellite6/hammer.py

```diff
@@ -427,3 +427,20 @@ def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
             'Wrong Activation key provided for determining CV and Env')
     return get_attribute_value(data, ak_name, 'content view'), \
         get_attribute_value(data, ak_name, 'lifecycle environment')
+
+
+def get_latest_cv_version(cv_name):
+    """Calculates the latest CV version to be published
+
+        :param string cv_name : Name of the CV for which version is to be
+        calculated
+        :return int : Calculated version to be created for CV
+        """
+
+    cv_version_data = hammer.hammer(
```

Comment:
Why hammer.hammer ? Only hammer is enough as you are using that function in same module.